lib/commands/description.js

```diff
@@ -42,19 +46,23 @@ if (typeof Program.locale === 'undefined') {
     process.exit(1);
 }
 
-
-// Run program
 const filePath = path.resolve(process.cwd(), Program.args[0]);
-const commander = new ShopwareStoreCommander(Program.username, Program.password);
-fs.exists(filePath)
-    .then((exists) => {
+(async () => {
+    try {
+        const password = await getPassword(Program);
+        // Run program
+        const commander = new ShopwareStoreCommander(Program.username, password);
+        spinner(commander.logger);
```

Comment:
You're passing an emitter here that is called `logger`? That's counter-intuitive.